packages/component/src/components/notification-center/index.tsx

```diff
@@ -235,8 +232,8 @@ function NotificationCard(props: NotificationCardProps): ReactElement {
       data-visible={isVisible}
       data-index={index}
       data-front={isFront}
-      data-swiping={swiping}
-      data-swipe-out={swipeOut}
+      data-swiping={false}
+      data-swipe-out={false}
```

Comment:
remove the swipe feature since it's silly to use a mouse to swipe the notification